packages/eslint-plugin/src/rules/jsx-pragma/README.md

```diff
@@ -0,0 +1,48 @@
+# @compiled/eslint-plugin/jsx-pragma
+
+This rule ensures a jsx pragma is used when using CSS prop.
+You can configure which jsx pragma to use via the `runtime` option (either classic or automatic).
```

Comment:
Can this be under an `## Options` heading, it's pretty typical in eslint rules. The format normally has each option under its own heading like this https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Reply:
Thanks will look at the examples